roles/nodejs/molecule/default/verify.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: check nodejs package status
+    package:
+      name: "{{ item }}"
+      state: present
+    check_mode: true
+    register: pkg_status
+    loop:
+      - nodejs
+      - yarn
+
+  - name: test for nodejs packages
```

Comment:
Should the assert be up with the loop?  Seems like this will just check of that last package is installed...
we used this is samba
```
    failed_when:
       - present is changed
```